lib/test-requirements.txt

```diff
@@ -14,8 +14,9 @@ plotly>=5.3.1
 psycopg2-binary>=2.9.1
 pydot>=1.4.2
 pyodbc>=4.0.32
+scipy<=17.3
```

Comment:
Sorry just missing a `.`  😄

Reply:
Fixed. Sorry! Kind of mia since later in the day